{"uuid": "1a184a8c-bf9d-4b94-9a6b-0acdfccf1b39", "children": ["5b5e2aa0-1a88-4416-84cd-d796f22b44ff", "f07a7264-4589-4e09-9c0f-4129aabae02e", "4a48a459-4409-4285-a28e-e15d07eef44c"], "befores": [{"name": "get_configuration", "status": "broken", "statusDetails": {"message": "KeyError: 'testbed-new'\n", "trace": "  File \"/usr/local/lib/python3.8/dist-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 1111, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.8/dist-packages/_pytest/fixtures.py\", line 883, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/imgd/ctpriv-scripts/altice/wlan-testing/tests/conftest.py\", line 373, in get_configuration\n"}, "start": 1696930138417, "stop": 1696930138417}], "start": 1696930138417, "stop": 1696930138511}